auth/models.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from sqlalchemy import Integer, Text
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from .. import db
+
+class User(db.Model):
+    __tablename__ = 'users'
+
+    id = mapped_column(Integer, db.Sequence('users_id_seq'), nullable=False, primary_key=True)
+
+    email = mapped_column(Text, unique=True, nullable=False)
```

Comment:
better create a `unique constraint` on email, since you can't have 2 users with same email.
on signup you should catch the exception thrown in such a case and return a proper error to the user.

Reply:
already exists